packages/manager/apps/telecom/src/app/telecom/pack/xdsl/access/pack-xdsl-access.controller.js

```diff
@@ -635,4 +657,74 @@ export default class XdslAccessCtrl {
         );
       });
   }
+
+  /**
+   * Check if change mail sending status is running
+   */
+  getChangeMailSendingTask() {
+    return this.XdslAccessService.getChangeMailTasks(
+      this.$stateParams.serviceName,
+    ).then(({ data }) => {
+      if (data.length > 0) {
+        this.isTaskRunning = true;
+        // Task change mail sending running
+        this.poller = this.$timeout(() => {
+          this.getChangeMailSendingTask();
+        }, POLLING_INTERVAL);
+      } else {
+        this.XdslAccessService.getAccess(this.$stateParams.serviceName).then(
+          (result) => {
+            const access = result.data;
+
+            // Set mail sending status
+            this.mailSendingStatus =
+              access?.mailSending || MAIL_SENDING_STATUS.noStatus;
+            this.mailSendingStatusLabel = this.$translate.instant(
+              `xdsl_detail_mail_sending_status_${this.mailSendingStatus}`,
+            );
+            this.isTaskRunning = false;
+          },
+        );
+      }
+    });
+  }
+
+  /**
+   * Change mail sending status
+   */
+  changeMailSending(status) {
+    this.mailSendingNewStatus = status;
+    this.mailSendingAsking = this.$translate(
+      status === this.MAIL_SENDING_STATUS.enabled
+        ? 'xdsl_detail_mail_sending_modal_enable'
```

Comment:
factorize prefix maybe?

Reply:
It's what @ghyenne asked me to do yesterday.

Reply:
Yes but it seems you could avoid repetition here of first part

Reply:
yes but it's the translation label not a variable or anything. We don't prefix all label for translation no?

Reply:
Yes it could be the value is enableD or disableD, not enable or disable.

Reply:
Exactly like this, thks
![image](https://github.com/user-attachments/assets/f6de7fa0-83d4-490b-87a2-1aa2b873b46b)